csharp_dotnetcore/18.Multi-Lingual-Bot/Translation/TranslationMiddleware.cs

```diff
@@ -0,0 +1,110 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
```

Comment:
There is a much richer (and more feature complete) version of this class - written by the Translation Team - that we should consider using. Check w/ Vishwac to see what the intent is of this class.